source/api_documentation/transactions_api/_rollup_message.rst

```diff
@@ -0,0 +1,13 @@
+.. raw:: html
+
+  <h2>
+    Signal Transactions Rollup
+  </h2>
+  <div class="alert alert-danger">
+    <p>
+      Starting with version <b>2019-12-01</b>, all future API versions will only return rollup rows of transaction types "Call", "Sale", and "Post Call Event".
```

Comment:
Can we try something simpler?
---------------------
API versions <b>2019-12-01</b> and later only return rollup rows of transaction types "Call", "Sale", and "Post Call Event".

Reply:
👍 
![image](https://user-images.githubusercontent.com/9993068/70750933-383f1180-1ce4-11ea-8f0b-26b059f61511.png)